services/src/main/java/org/exoplatform/officeonline/rest/WOPIResource.java

```diff
@@ -484,39 +488,46 @@ public Response files(@Context UriInfo uriInfo,
       }
     }
     case REFRESH_LOCK: {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("WOPI Request handled: refreshLock");
-      }
       String providedLock = request.getHeader(LOCK);
       return refreshLock(config, providedLock);
     }
     case RENAME_FILE: {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("WOPI Request handled: renameFile");
-      }
       String name = request.getHeader(REQUESTED_NAME);
       String lock = request.getHeader(LOCK);
       return renameFile(fileId, config, name, lock);
     }
     case UNLOCK: {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("WOPI Request handled: unlock");
-      }
       String providedLock = request.getHeader(LOCK);
       return unlock(config, providedLock);
     }
     case DELETE: {
-      if (LOG.isDebugEnabled()) {
-        LOG.debug("WOPI Request handled: delete");
-      }
       String lock = request.getHeader(LOCK);
       return delete(config, lock);
     }
+    case PUT_USER_INFO: {
+      try {
+        String userInfo = convertStreamToString(request.getInputStream());
+        return putUserInfo(config, userInfo);
+      } catch (IOException e) {
+        return Response.status(Status.INTERNAL_SERVER_ERROR)
```

Comment:
Seems we need LOG.error with related information here.

Reply:
Fixed: https://github.com/exo-addons/office-online/pull/14/commits/ca31f5a28a2aca8b055bb63bea9c00c314d8870a